Modules/UserDefaults/README.md

```diff
@@ -0,0 +1,12 @@
+#  UserDefaults
+
+The user defaults module provides a property wrapper to access data in the device user defaults storage.
+
+Example:
+
+```swift
+struct MyDataRepository {
+    @UserDefault(key: "<#value identifier#>", defaultValue: true)
```

Comment:
Maybe you could adjust the example in the way that a user defaults instance can be passed.